source/_integrations/climate.modbus.markdown

```diff
@@ -66,6 +66,11 @@ data_type:
   required: false
   type: string
   default: float
+structure:
+  description: "If data_type is custom specify here a double quoted Python struct format string to unpack the value. See Python documentation for details. Ex: >i."
```

Comment:
```suggestion
  description: "If `data_type` is custom specified a double-quoted Python struct is expected here, to format the string to unpack the value. See Python documentation for details. Example: `>i`."
```

Reply:
Hi @frenck, I copied a whole block from the docs for `Modbus sensor`. Can I fix the line there as well as part of this PR?

Reply:
I updated this description also for `Modbus sensor`. It's only a cosmetic change. Please let me know if I should split this into a separate PR. Thanks!